tests/search_current_dir/base_dir_with_spaces.fish

```diff
@@ -0,0 +1,8 @@
+mock commandline --current-token "echo tests/_resources/multi word dir/"
+mock commandline "--current-token --replace --" "echo \$argv"
+mock commandline \* ""
+set --export --append FZF_DEFAULT_OPTS "--filter=''" # automatically select all input lines sent to fzf
+
+set actual (__fzf_search_current_dir)
+set expected "'tests/_resources/multi word dir/file 1.txt' 'tests/_resources/multi word dir/file 2.txt'"
+@test "uses current token as base directory if it ends in / and is a directory" "$actual" = $expected
```

Comment:
Just realized we never really fixed bug 1.

![image](https://user-images.githubusercontent.com/44045911/114277600-61012380-9a5e-11eb-96ea-5918936880d0.png)

Reply:
How would you modify the test to catch this new case? I don't understand why it wasn't caught.

Reply:
I'm unable to repro this

Reply:
> How would you modify the test to catch this new case? I don't understand why it wasn't caught.

Not to my knowledge, sorry. May take a look when I get the time. 

> I'm unable to repro this

I can confirm it's not related to my shell profile. Cam you reproduce with `mkdir 'with spaces'`, type `./with\ spaces` and trigger the function?

Reply:
Nvm I see that the test doesn't test the preview function, which I don't know how to do.

I can repro now. I made a mistake before. Thanks for reporting this!

Reply:
Crazy I made the PR about fixing this bug but I didn't actually fix it 🤦‍♂️

Reply:
> Crazy I made the PR about fixing this bug but I didn't actually fix it 🤦‍♂️

Well I was quite confused I didn't notice it either back then.

A good lesson on always testing changes manually if possible 😄

Reply:
> Nvm I see that the test doesn't test the preview function, which I don't know how to do.

Maybe replace the preview function _file_ with a test function, which tests if the command output is expected.

Reply:
> Maybe replace the preview function file with a test function, which tests if the command output is expected.

I can't get fzf to execute the search function in scripted mode. It'd have to not exit automatically, but if it doesn't exit automatically, how do I make it exit so that the test can continue to run.

Reply:
Start a background shell, sleep for a few seconds and kill all `fzf` processes?

Reply:
Thanks for the idea. I might use it if depending on what happens to this issue https://github.com/junegunn/fzf/issues/2442.